k-distribution/tests/regression-new/freshConfig/test.k

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) K Team. All Rights Reserved.
```

Comment:
Can we name the folder `issue-3520-freshConfig`?
So that we maintain some sort of convention. It could make it easier in the future to track down changes.